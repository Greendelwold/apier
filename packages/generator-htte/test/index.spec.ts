import Generator from '../src';
import { loadApiers } from "@dee-contrib/apier-test-utils-generator";

test('generate', () => {
  const apier = loadApiers('general')[0];
  const generator = new Generator(apier);
  expect(generator.value).toMatchSnapshot();
});
