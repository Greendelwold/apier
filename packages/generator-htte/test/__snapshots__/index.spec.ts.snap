// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generate 1`] = `
Object {
  "describe": "获取模型",
  "req": Object {
    "body": Object {
      "array": Array [
        "1",
        "2",
      ],
      "bool": true,
      "integer": 32,
      "null": null,
      "number": 16.23,
      "object": Object {
        "foo": 3,
      },
    },
    "headers": Object {
      "X-ORG-ID": 32432,
    },
    "method": "get",
    "params": Object {
      "id": 32234,
    },
    "query": Object {
      "pageNo": 1,
      "pageSize": 10,
    },
    "url": "/model/{id}",
  },
  "res": Object {
    "body": Object {
      "array": Array [
        Object {
          "foo": 1,
        },
        Object {
          "bar": 2,
        },
      ],
      "bool": true,
      "integer": 32,
      "null": null,
      "number": 16.23,
      "object": Object {
        "password": "a234324",
      },
    },
    "status": 200,
  },
}
`;
